docs/src/markdown/extensions/magiclink.md

```diff
@@ -479,6 +480,51 @@ and will copy them for your custom repository. If this is not sufficient, you ca
 `shortener_user_exclude` for your custom repository provider using your specified `name`. If you manually set excludes
 in this manner, no excludes from the same `type` will be copied over.
 
+## Simple User-Defined References
```

Comment:
@facelessuser - When you get a chance to take a look, here is proposed documentation for `simplerefs` which we can consider a spec for the feature.